# HG changeset patch
# Date 1304363258 -10800
# User Mihai Sucan <mihai.sucan@gmail.com>
# Parent bc1cd5fc588510323c630f5e1d68a0896fb37114
add Erik Vold to the contributors list for workspace.xul

diff --git a/browser/base/content/workspace.xul b/browser/base/content/workspace.xul
--- a/browser/base/content/workspace.xul
+++ b/browser/base/content/workspace.xul
@@ -18,16 +18,17 @@
    - The Initial Developer of the Original Code is
    - The Mozilla Foundation.
    - Portions created by the Initial Developer are Copyright (C) 2011
    - the Initial Developer. All Rights Reserved.
    -
    - Contributor(s):
    -   Rob Campbell <robcee@mozilla.com> (original author)
    -   Mihai Sucan <mihai.sucan@gmail.com>
+   -   Erik Vold <erikvvold@gmail.com>
    -
    - Alternatively, the contents of this file may be used under the terms of
    - either the GNU General Public License Version 2 or later (the "GPL"), or
    - the GNU Lesser General Public License Version 2.1 or later (the "LGPL"),
    - in which case the provisions of the GPL or the LGPL are applicable instead
    - of those above. If you wish to allow use of your version of this file only
    - under the terms of either the GPL or the LGPL, and not to allow others to
    - use your version of this file under the terms of the MPL, indicate your
